### DISCUSSION TITLE
How do I change my GitHub Student Developer Pack email address

### DISCUSSION THREAD

**Discussion by amakerlife** [community/community#40418] [GitHub Education] [answered]
The Student Pack email is [ml@makerlife.ml](mailto:ml@makerlife.ml). But I choose my personal email [yuyanggu@hotmail.com](mailto:yuyanggu@hotmail.com) to register other apps. I want to swap Student Pack email to [yuyanggu@hotmail.com](mailto:yuyanggu@hotmail.com) (GitHub account main email) . How can I do?

**Comment by SilasRodrigues19** [comment]
@amakerlife Hi.

In [this link](https://education.github.com/discount_requests/pack_application) you can change the email. But why do you want to do this? You can log in with all the emails registered on your GitHub, which you can check at [this link
](https://github.com/settings/emails)

**Reply by amakerlife** [reply]
> @amakerlife Hi.
> 
> In [this link](https://education.github.com/discount_requests/pack_application) you can change the email. But why do you want to do this? You can log in with all the emails registered on your GitHub, which you can check at [this link ](https://github.com/settings/emails)

Hi,

Oops, I cannot change the email in that link because I have registered. 

![](https://img.makerlife.tk/data/20221202103248.png)

When I signed up for Azure, I chose the hotmail, because this is my Microsoft personal account, but I used the makerlife amail to apply for the GitHub Student Developer Pack, so Azure told me that the email has no student pack.

**Reply by rekoer** [reply]
Hi, same problem here. I‘m looking for a way to use an other mail address because my used stundet-mail is already in an Azure account of my school. So Azure student benefits are not working with that address.

**Reply by amakerlife** [reply]
> Hi, same problem here. I‘m looking for a way to use an other mail address because my used stundet-mail is already in an Azure account of my school. So Azure student benefits are not working with that address.

Hi, if you find a way, you can @me and tell me what can I do. I have contacted GitHub but no one has responded to me yet.

**Comment by Pavan-Nambi** [comment]
I know this is late but you can change emails from settings right , i changed like that and i didnt faced any issues...

**Reply by harshpreet14** [reply]
so, did you still have the student developer pack? because you initially registered on the previous mail id

**Reply by amakerlife** [reply, marked as answer]
You can ask the GitHub Support to change your Email address. They will cancel your pack and you can request again
